docker-compose-dev.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.7'
+
+services:
+  web:
+    restart: always
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: dev
+    entrypoint: ./app/entrypoint.sh
+    volumes:
+      - ./static_volume/:/staticfiles
+      - ./media_volume/:/mediafiles
```

Comment:
why need these volumes?

Reply:
static for admin panel, media can be deleted

Reply:
done